.travis.yml

```diff
@@ -82,7 +82,7 @@ jobs:
   #   - EXTRA_ARGS=
 
 install:
-- pip install -U pip setuptools
+- pip install -U pip==20.3.4 setuptools
```

Comment:
nit: could you add a comment that mentions Python 3.5 in some way so this is discoverable via grep

Reply:
A good idea, done.